web/src/main/java/org/akaza/openclinica/controller/openrosa/processor/FSItemProcessor.java

```diff
@@ -277,6 +281,26 @@ private void processFieldSubmissionGroupItems(ArrayList<HashMap> listOfUploadFil
                 }
             } else {
                 logger.error("Failed to lookup item: '" + itemName + "'.  Continuing with submission.");
+                int lastIndexOf = itemName.lastIndexOf(REPEATCOUNT);
```

Comment:
This method is getting really really big. Is it possible to break it down? At least the newly added code in a different method?